scratchdata/scratchdata
# refactor router.go

This is involved, made sure testing all paths was done:

```
- testing
	- edit connection
		- new api key
		- delete
	- connect database
		- duckdb
		- clickhouse
			- couldn't verify it worked
		- redshift
		- bigquery
			- error when inserting data, noted in slack
		- test inserting data via api and querying
	- request connection
		- http://localhost:8080/dashboard/request/9b3495a4-ee7c-43b9-912c-a577efbf8237 -> unauth session
		- intentionally error
		- success
        - data sharing
	    - download
```
